dxr/testing.py

```diff
@@ -342,12 +334,12 @@ class DxrInstanceTestCaseMakeFirst(DxrInstanceTestCase):
     """
     @classmethod
     def generate(cls):
-        check_call(['make'], cwd=join(cls.this_dir(), 'code'))
         super(DxrInstanceTestCaseMakeFirst, cls).generate()
+        check_call(['make'], cwd=join(cls._config_dir_path, 'code'))
 
     @classmethod
     def teardown_class(cls):
-        check_call(['make', 'clean'], cwd=join(cls.this_dir(), 'code'))
+        check_call(['make', 'clean'], cwd=join(cls._config_dir_path, 'code'))
```

Comment:
I guess it doesn't hurt to keep calling `make clean` for now. Who knows what people do in those makefiles? They could be ickily doing something outside the test dir.

Reply:
Yes.  I just didn't really see any need to change this.  It shouldn't cause any problem to keep it and if there is really some external cleanup that needs to be done we'll still perform it.